pkgs/tools/security/truecrack/default.nix

```
@@ -0,0 +1,40 @@
+{ lib, gccStdenv, fetchFromGitLab, cudaSupport, cudatoolkit,
+pkg-config }:
```

should probably provide the default here
```suggestion
{ lib, gccStdenv, fetchFromGitLab, cudatoolkit
, cudaSupport ? false
, pkg-config }:
```